docs/mig.md

```diff
@@ -0,0 +1,206 @@
+# vGPU/MIG configuration
+
+This page details how to configure Multi Instance GPU (MIG) in Slurm.
+
+## Pre-requisites
+
+- Image built with cuda support. This should automatically recompile slurm against NVML.
```

Comment:
Why can't I see the task `Recompile and install slurm packages` in any checks for this PR?

Reply:
See here: https://github.com/stackhpc/ansible-slurm-appliance/actions/runs/15183034343/job/42696786826?pr=656#step:10:23240